doc/modules/computing.rst

```diff
@@ -290,14 +292,14 @@ interesting, but for many applications we would better not increase
 prediction latency too much. We will now review this idea for different
 families of supervised models.
 
-For :mod:`sklearn.linear_model` (e.g. Lasso, ElasticNet,
+For :mod:`linear_model` (e.g. Lasso, ElasticNet,
```

Comment:
To be consistent with `doc/faq.rst` I removed the `sklearn.` part, however I broke the link with this.